src/request/components/request-detail-panel-execution.jsx

```diff
@@ -397,6 +455,13 @@ module.exports = React.createClass({
                 preCommands = <CommandList beforeExecuteCommandMessages={beforeExecuteCommandMessages} afterExecuteCommandMessages={afterExecuteCommandMessages} mongoDBMessages={mongoDBMessages} beginMessage={beginRequestMessage} endMessage={beforeActionInvokedMessage} isRoot={true} />            
             }
             
+            // process middleware
+            var middleware = null;
+            
```

Comment:
Yes, I was just trying to follow the pattern/convention defined by the larger function.  I've removed it.